src/assets/scss/_gallery.scss

```diff
@@ -0,0 +1,196 @@
+//Gallery section
+
+.gallery {
+    display: flex;
+    align-items: center;
+    padding: 8rem 10rem;
+    flex-wrap: wrap;
+    height: 45rem;
+    padding-right: 0;
+
+    @media (max-width: $mid-size){
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+    }
+
+    @media (max-width: $mobile-size) {
+      height: 30rem;
+    }
+  
+    
+    &__btn-container {
+      display: flex;
+      width: 19rem;
+      justify-content: space-between;
+      position: absolute;
+      margin-top: 27rem;
```

Comment:
to bardzo duży margin-top, może parent powinnien mieć position relative, a jak nie ma parenta to może warto go stworzyć tylko po to, żeby miał jakąś wysokość i position relative :)